PyInstaller/hooks/hook-speech_recognition.py

```diff
@@ -0,0 +1,14 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+# Hook for speech_recognition: https://pypi.python.org/pypi/SpeechRecognition/
```

Comment:
@Uberi Could you please add comments about what version and what OS this hook is tested with?